vignettes/baselinenowcast.Rmd

```diff
@@ -274,12 +282,60 @@ The reporting triangle we are applying it to must have the same `max_delay`
 as the delay estimate.
 
 ```{r}
-# point_reporting_square <- apply_delay( #nolint
-#   triangle_to_nowcast = triangle,#nolint
-#   delay_df#nolint
-# )#nolint
+point_reporting_square <- apply_delay(
+  triangle_to_nowcast = triangle,
+  delay_pmf = delay_df$pmf
+)
 ```
 
+We'll make a quick plot to compare the nowcasted confirmed cases through
+July 1, 2021, from the observations up until October 1, 2021.
+We'll compare this to the right-truncated data available up until July 1, 2021.
+```{r}
+final_data <- epinowcast::germany_covid19_hosp[location == "DE"][age_group == "00+"] |> # nolint
+  enw_filter_report_dates(latest_date = "2021-10-01") |>
+  enw_filter_reference_dates(
+    latest_date = "2021-07-01",
+    include_days = n_history - 1
+  ) |>
+  group_by(reference_date) |>
+  summarise(
+    total_confirmed = max(confirm)
+  ) |>
+  dplyr::mutate(nowcast = rowSums(point_reporting_square))
+
+
+ggplot() +
+  geom_line(
+    # Plot the data summed across reporting days as of July 1,2021
+    data = observed_long |>
+      group_by(reference_date) |>
+      summarise(total_confirmed = max(confirm)),
+    aes(x = reference_date, y = total_confirmed), color = "darkred"
+  ) +
+  geom_line(
+    data = final_data,
+    aes(x = reference_date, y = total_confirmed), color = "black"
+  ) +
+  geom_line(
+    data = final_data,
+    aes(x = reference_date, y = nowcast), color = "darkblue"
+  ) +
+  theme_bw() +
+  xlab("Reference date") +
+  ylab("Confirmed admissions") +
+  scale_y_continuous(trans = "log10") +
+  ggtitle("Comparing real-time, nowcasted, and later observed cases")
+```
+Here we can see that our point nowcast slightly overestimates what was
+eventually reported (black line), but does a decent overall job of correcting
+for the right-truncation observed in the red line (the data prior to the
+nowcast).
+
+What we of course really want is a probabilistic estimate of the nowcasted
```

Comment:
why of cours? I am new to this whole nowcasting thing and I just want my dashboard to look cool

Reply:
removed